content/2015.10.01.outreachy.rst

```diff
@@ -0,0 +1,120 @@
+Looking for an outreachy intern this winter
+###########################################
+
+:slug: outreachy-2016
+:date: 2015-10-01
+:lang: en
+:url: outreach-kinto-2016
+:summary:
+    This year, we are participating to the Gnome outreachy program with our
+    Kinto project, in order to make instances discoverable. Please, join in!
+
+.. epigraph::
+
+  By having a program targeted specifically towards women, we found that we
+  reached talented and passionate participants, who were uncertain about how to
+  start otherwise.
+
+  We are now expanding the program to people from more groups underrepresented
+  in FOSS. We hope this effort will help many people learn how exciting, varied
+  and valuable work on FOSS projects can be and how inclusive the community
+  really is.
+
+  -- `The Gnome outreachy program wiki <https://wiki.gnome.org/Outreachy>`_
+
+In our team, and at Mozilla in general, we thrive to build an inclusive
+community. And we believe being more inclusive starts by reaching to the people
+who are often under-represented in tech.
+
+That's why we are participating in the gnome Outreachy program this year by
+proposing an internship around Kinto and making instances discoverable.
+
+Check out the eligibility details at the end of the article if you're
+interested!
+
+Making Kinto instances discoverable
+===================================
+
+The `Kinto project <https://kinto.readthedocs.org>`_ aims to bring storage
+instances to everyone, attached to their
+`Firefox Accounts <https://accounts.firefox.com>`_ (Well, it actually supports
+multiple authentication schemes, but FxA is integrated with it, and that is
+part of the solution we want to deliver).
```

Comment:
FxA is not integrated with it, is it?

I propose : *Well, it actually supports multiple authentication schemes, but we want to promote Firefox Accounts as a first step*. Thoughts ?